Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Multiple-Textbox/index.md

```diff
@@ -50,4 +50,46 @@ The Repeatable textstrings property editor enables a content editor to make a li
         </ul>
     }
 }
-```
\ No newline at end of file
+```
+
+## Add values programmatically
+
+See the example below to see how a value can be added or changed programmatically. To update a value of a property editor you need the [Content Service](../../../../../Reference/Management/Services/ContentService/index.md).
+
+```csharp
+@{
+    // Get access to ContentService
+    var contentService = Services.ContentService;
+
+    // Create a variable for the GUID of the page you want to update
+    var guid = new Guid("32e60db4-1283-4caa-9645-f2153f9888ef");
+
+    // Get the page using the GUID you've defined
+    var content = contentService.GetById(guid); // ID of your page
+
+    // Set the value of the property with alias 'keyFeatureList'
+    content.SetValue("keyFeatureList", "Awesome" + Environment.NewLine + "Super");
```

Comment:
`Environment.NewLine` - I'm not familiar with this - is that what you need to add between all the text strings added, like, in order to separate them?

Reply:
@sofietoft Yes. To add multiple values to the repeatable textstrings property editor you have to put each value on a new line. So basically: `Awesome \r\n Super \r\n Nice` . `Environment.NewLine` does this for you.` \r\n` for non-Unix platforms, or `\n` for Unix platforms. See the Microsoft docs as well https://docs.microsoft.com/en-us/dotnet/api/system.environment.newline?view=net-5.0 Does this make sense?

Reply:
Ah! Yes, that makes perfect sense 😎 

Should we perhaps add a little note below the code snippet, about that? 🤔

Reply:
@sofietoft Sounds good to me, I added a note 😎